src/modules/callouts/pages/CalloutsAdminPage.vue

```diff
@@ -32,12 +32,12 @@
           <AppItemStatus :status="value" />
         </template>
         <template #title="{ item, value }">
-          <a
-            :href="'/tools/polls/' + item.slug"
+          <router-link
+            :to="'/admin/callouts/' + item.slug"
```

Comment:
Also needs to change [here](https://github.com/beabee-communityrm/beabee-frontend/blob/94239ec714f16b77026da310a00929af1d89d5ff/src/modules/callouts/pages/CalloutsAdminPage.vue#L8-L10)